src/main/scala/BrokenKeyboard.scala

```diff
@@ -0,0 +1,33 @@
+object BrokenKeyboard extends App {
+
+  def getWord(readlines:String) {
+    val source = io.Source.fromFile("Users/Admin/Documents/DijanR/scala/enable1.txt")
+    for(item <- source)
+    for (r <- 0 until readlines.length) yield (r)
```

Comment:
if I get understand it correctly you use these two for loops to get all the words from the file, you could make it all in one line like this Source.fromFile("words.txt").getLines.toList